lciafmt/iw.py

```diff
@@ -40,21 +50,93 @@ def _read(access_file: str) -> pandas.DataFrame:
 
     cnxn = pyodbc.connect(connStr)
     crsr = cnxn.cursor()
-    crsr.execute("SELECT * FROM [CF - not regionalized - All other impact categories]")
-    rows = crsr.fetchall()
+
+# List relevant sheets in Impact World Access file. Second item in tuple tells the source of compartment information.
+# Compartment for water categories are not included in access file, defined below.
+# Elementary flow names are used to define the compartment for land transformation and occupation
+# Compartment and Subcompartment data is available in the Access file for other categories.
+    Regional_sheets = [("CF - regionalized - WaterScarc - aggregated", "Raw/in water"),
+                       ("CF - regionalized - WaterAvailab_HH - aggregated", "Raw/in water"),
+                       ("CF - regionalized - LandTrans - aggregated", "Elementary Flow"),
+                       ("CF - regionalized - LandOcc - aggregated", "Elementary Flow"),
+                       ("CF - regionalized - EutroMar - aggregated", "Compartment"),
+                       ("CF - regionalized - PartMatterForm - aggregated","Compartment"),
+                       ("CF - regionalized - AcidFW - aggregated", "Compartment"),
+                       ("CF - regionalized - AcidTerr - aggregated", "Compartment"),
+                       ("CF - regionalized - EutroFW - aggregated", "Compartment"),
+                       ("CF - not regionalized - All other impact categories", "Compartment")]
 
     records = []
-    for row in rows:
-        df.record(
-            records,
-            method="Impact World",
-            indicator = row[1],
-            indicator_unit=row[2],
-            flow=row[5],
-            flow_category=row[3] + "/" + row[4],
-            flow_unit=row[8],
-            cas_number=util.format_cas(row[6]).lstrip("0"),
-            factor=row[7])
+
+    for x in Regional_sheets:
```

Comment:
where possible, variable names should be lower case as capital case often indicates a Class
see the PEP style guidelines: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names